### PR TITLE
update cli scenarios

### PR DESCRIPTION
**Pull request by briacht**
## Summary

add image classification to cli scenarios

### CODE REVIEW

File: docs/machine-learning/automate-training-with-cli.md [hunk 1]
```diff
@@ -28,10 +28,11 @@ You can generate those assets from your own datasets without coding by yourself,
 
 Currently, the ML Tasks supported by the ML.NET CLI are:
 
-- classification (binary and multi-class)
+- classification
 - regression
 - recommendation
-- Future: other machine learning tasks such as image-classification, ranking, anomaly-detection, clustering
+- image classification
+- Future: other machine learning tasks such as ranking, anomaly detection, clustering
```
Comment (luisquintanilla) [inline docs/machine-learning/automate-training-with-cli.md:35]: I wonder if we even need this future bullet point. It's not in the product now, so would remove it and just list what the product supports. Sounds more like a promise and might create confusion.

**Review by luisquintanilla — COMMENTED** [review]
@briacht thanks for updating this. 

I added a comment. Let me know what you think.

File: docs/machine-learning/automate-training-with-cli.md (hunk 1 above)
Comment (gewarren) [reply docs/machine-learning/automate-training-with-cli.md:35]: Yes, docs shouldn't mention future plans according to the contributor's guide. @briacht Could you update and then we'll merge?

**Review by gewarren — CHANGES_REQUESTED** [review]

**Comment by adegeo** [comment]
@briacht Ping

**Comment by briacht** [comment]
I had to create a new PR: #23283

Also I got rid of the "Future" statement.

**Comment by luisquintanilla** [comment]
Closing PR since there's a new one.

**Pull request closed without merging**